src/coreclr/src/utilcode/makepath.cpp

```diff
@@ -101,7 +101,7 @@ void MakePath (
                 }
 
 #ifdef _MBCS
```

Comment:
Is `_MBCS` ever defined? Delete this instead?

Reply:
`_MBCS` and `_ismbblead` another unused ~1000 loc, which is removed in ba49728.